src/views/AuthorityView/AuthorityView.js

```diff
@@ -59,31 +65,44 @@ const AuthorityView = ({
     });
   };
 
+  const hasEditPermission = () => {
+    const { stripes } = PropTypes;
+
+    return stripes.hasPerm('ui-marc-authorities.authority-record.edit');
+  };
+
   return (
-    <MarcView
-      paneTitle={authority.data.headingRef}
-      paneSub={intl.formatMessage({
-        id: 'ui-marc-authorities.authorityRecordSubtitle',
-      }, {
-        heading: authority.data.headingType,
-        lastUpdatedDate: intl.formatDate(marcSource.data.metadata.lastUpdatedDate),
-      })}
-      isPaneset={false}
-      marcTitle={intl.formatMessage({ id: 'ui-marc-authorities.marcHeading' })}
-      marc={marcSource.data}
-      onClose={onClose}
-      lastMenu={(
-        <IfPermission perm="ui-marc-authorities.authority-record.edit">
-          <Button
-            buttonStyle="primary"
-            marginBottom0
-            onClick={redirectToQuickMarcEditPage}
-          >
-            <FormattedMessage id="ui-marc-authorities.authority-record.edit" />
-          </Button>
-        </IfPermission>
-      )}
-    />
+    <KeyShortCutsWrapper
+      onEdit={redirectToQuickMarcEditPage}
+      isPermission={hasEditPermission}
+    >
+      <div data-testid="provider-content">
```

Comment:
Do we need this `div` here? If no, please remove it. If yes, change the `data-testid` because now it doesn't match the content.